config/main.py

```diff
@@ -14,28 +14,46 @@
 import ipaddress
 from swsssdk import ConfigDBConnector, SonicV2Connector, SonicDBConfig
 from minigraph import parse_device_desc_xml
+from config_mgmt import configMgmt
```

Comment:
we are planning to convert class name to PascalCase so it will become ConfigMgmt

Reply:
#done